.env-advanced.example

```diff
@@ -1,3 +1,6 @@
 # Advanced configuration. Normally, it shouldn't be modified
-ETH_NETWORK=mainnet
+ETH_NETWORK=ropsten
+ALLOW_LEGACY_ENTITIES=true
+LOG_LEVEL=debug
+METRICS=true
```

Comment:
Are we sure we want to enable metrics for all catalysts by default? Shouldn't that be opt-in?

Reply:
Yes, it should

Reply:
It is anyways a .example file

Reply:
I assume most people will just copy the example file, so maybe we could set it to `false`?

Reply:
cool